Game/include/Scripts/LapCheckpointScript.h

```diff
@@ -0,0 +1,32 @@
+#pragma once
+
+#include "Script.h"
+#include "LapManagerScript.h"
+#include "../../../Physics/include/Aabb.h"
+
+class LapCheckpointScript : public Script
+{
+public:
+	LapCheckpointScript(Entity* mEntity);
+	~LapCheckpointScript();
+
+	void Init(LapManagerScript* lapManager, TransformComponent* other, int checkpointIndex);
```

Comment:
Change and override to SetParameters().
Conform to Script.h format and extend ScriptParameters struct.
Use OrbitScript for reference.